src/components/Inputs/ComboBox/ComboBoxMenuItem.tsx

```diff
@@ -79,13 +84,17 @@ export const ComboBoxMenuItem = <T extends ComboBoxOptionRequired>(
   const handleOnClick = (event: MouseEvent) => {
     event.stopPropagation();
     onItemSelect(item);
+
+    if (!('values' in props) && !multiselect) return;
```

Comment:
`!('values' in props)` is the same as `('value' in props)`  I think?

Reply:
@mariush2 Typescript doesn't "understand" that logic :( 
![image](https://github.com/equinor/amplify-components/assets/23264453/3d8c57c3-221d-4d82-b40c-2cca97fb7b5e)